ios/Classes/SwiftCustomerIoPlugin.swift

```diff
@@ -119,6 +127,28 @@ public class SwiftCustomerIoPlugin: NSObject, FlutterPlugin {
         CustomerIO.shared.profileAttributes = attributes
     }
     
+    private func registerDeviceToken(params : Dictionary<String, AnyHashable>){
+        guard let token = params[Keys.Tracking.token] as? String
+        else {
+            return
+        }
+        
+        CustomerIO.shared.registerDeviceToken(token)
```

Comment:
I ain't an iOS expert 😅 But doesn't if statement looks better here? e.g.
```
if let token = params[Keys.Tracking.token] as? String {
    CustomerIO.shared.registerDeviceToken(token)
}
```
Mainly because I think the method is too small for early returns.

Reply:
haha! I don't have any preference but I saw this being followed in RN iOS, so kept the same format. 😄